mod_xsendfile.c

```diff
@@ -292,8 +295,19 @@ static apr_status_t ap_xsendfile_get_filepath(request_rec *r,
   return rv;
 }
 
+static int ap_xsendfile_add_request_header(request_rec *r) {
+  xsendfile_conf_active_t enabled = ((xsendfile_conf_t *)ap_get_module_config(r->per_dir_config, &xsendfile_module))->enabled;
```

Comment:
This is duplicated code, add some function like `ap_xsendfile_enabled_for(request_rec*)` and use it in all places.